### ISSUE TITLE
Keep screenshots on the website and in the README

### ISSUE THREAD

**Issue by hoijui** [mahogny/docubricks#14]
People interested in using a visual editor want to see something visual.

**Comment by MakerTobey** [comment]
Good point. This is an easier version of video tutorials which have been on meh ToDo-list for a long time. It's on my list now.

**Comment by hoijui** [comment]
ahhh... video tutorials are great! :-)
lots of work.. screenshots are easy. ;-)
but cool! ride on!
